### PR TITLE
update classnames

### PR DESCRIPTION
**Pull request by irmakyz**

### CODE REVIEW

File: src/styles/elements/GeoCalendar/_geo-calendar.scss [hunk 1]
```diff
@@ -64,14 +64,8 @@
     justify-content: space-evenly;
     padding-bottom: $space-base;
   }
-
-  .geo-calendar__input {
```
Comment (Navas23) [inline src/styles/elements/GeoCalendar/_geo-calendar.scss:68]: Did we remove these styles?

File: CHANGELOG.md [hunk 2]
```diff
@@ -4,10 +4,31 @@ Changed:
...
 - `geo-editable-input__container__buttons`: `geo-editable-input__buttons-container`
-`GeoDropdownRegularButton`: The classnames are renamed according to BEM notation:
+- `GeoDropdownRegularButton`: The classnames are renamed according to BEM notation:
 - `geo-dropdown__regular-button-container`: `geo-dropdown-regular-button-container`
 - `geo-dropdown__regular-button-container__icon`: `geo-dropdown-regular-button-container__icon`
+- `GeoCalendar`: The classnames are renamed according to BEM notation:
 - `geo-calendar__input__date-feedback--error`: `geo-calendar-input__date-feedback--error`
+- `GeoFeedbackBox`: The classnames are renamed according to BEM notation:
+- `geo-feedback-box__content`: `geo-feedback-box-content`
+- `geo-feedback-box__content__icon`: `geo-feedback-box__content__icon`
```
Comment (Navas23) [inline CHANGELOG.md:14]: Review these classes, they are not correct

File: CHANGELOG.md [hunk 3]
```diff
@@ -4,10 +4,31 @@ Changed:
...
+- `geo-feedback-box__content__close-icon`: `geo-feedback-box__content__close-icon`
+- `geo-feedback-box__content__body`: `geo-feedback-box__content__body`
+- `geo-feedback-box__content__actions`: `geo-feedback-box__content__actions`
+- `geo-segmented-control-item--info--active`: `geo-segmented-control-item--info active`
```
Comment (Navas23) [inline CHANGELOG.md:18]: This is not what we want, right? having an "active" separated class does not make sense

**Review by Navas23 — CHANGES_REQUESTED** [review]

File: src/elements/GeoSegmentedControl/GeoSegmentedControlItem.vue [hunk 4]
```diff
@@ -1,12 +1,11 @@
 <template>
   <geo-trimmed-content
     :class="{
-      [`geo-segmented-control-item${outlineSuffix}`]: true,
-      [`geo-segmented-control-item--${variant}${outlineSuffix}`]: true,
-      [`geo-segmented-control-item${outlineSuffix}--active`]: active,
-      [`geo-segmented-control-item--${variant}${outlineSuffix}--active`]: active,
-      [`geo-segmented-control-item${outlineSuffix}--disabled`]: disabled,
-      [`geo-segmented-control-item--${variant}${outlineSuffix}--disabled`]: disabled
+      [`geo-segmented-control-item${outlineSuffix(true)}`]: true,
```
Comment (Navas23) [inline src/elements/GeoSegmentedControl/GeoSegmentedControlItem.vue:4]: If we have the suffix here... why do we need it on the rest?

**Review by Navas23 — APPROVED** [review]

**Review by dletran — APPROVED** [review]

**Pull request merged**